src/platforms/k8s.ts

```diff
@@ -0,0 +1,70 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+// tslint:disable:object-curly-spacing
+
+import { Command } from '@oclif/command'
+import * as commandExists from 'command-exists'
+import * as execa from 'execa'
+import * as Listr from 'listr'
+
+export class K8sHelper {
+  startTasks(flags: any, command: Command): Listr {
+    return new Listr([
+      {
+        title: 'Verify if kubectl is installed',
+        task: () => {
+          if (!commandExists.sync('kubectl')) {
+            command.error('E_REQUISITE_NOT_FOUND')
+          }
+        }
+      },
+      { title: 'Verify remote kubernetes status',
+        task: async (_ctx: any, task: any) => {
```

Comment:
Could we use something like https://github.com/che-incubator/chectl/blob/master/src/commands/server/stop.ts#L59 to avoid some adherence to `kubectl`?

Reply:
yes, done

Reply:
BTW I still check for kubectl tool as it's used sometimes in start sequences depending on the installer